styles/stylesheet.css

```diff
@@ -0,0 +1,377 @@
+
+}
+
+.container-blog-home img {
+	margin-left: 10%;
+	height: auto;
+	width: 50%;
+	
+	
+}
+
+
+.container-blog-home h3 {
+	color: #116694;
+	font-family: 'Arvo', serif;
+	font-size: 1.5rem;
+	font-weight: bold;
+	position: relative;
+}
+
+.container-blog-home p {
+	color: black;
+	font-family: 'Arvo', serif;
+	font-size: 1.5rem;
+	position: relative;
+}
+
+.blog-post-home {
+	margin-left: 10%;
+}
+
+.blog-post-home h3 {
+	padding: 0.5rem;
+}
+
+.blog-post-home p {
+	font-size: 1rem;
+}
+
+.blog-post-home a {
+	color: #00BFFF;
+	font-family: 'Arvo', serif;
+	font-size: 1.2rem;
+	font-weight: bold;
+	position: relative;
+	text-decoration: none;
+}
+
+.blog-home-block {
+	border-radius: 25px;
+}
+
+
+.grid-container {
+	display: grid;
+	margin-left: 10%;
+	grid-template-columns: auto auto auto auto;
+	grid-column-gap: 2rem;
+	padding: 0.6rem;
+	margin-right: 10%;
+	grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+}
+.grid-item {
+	max-width: 100%;
+	height: auto;
+	text-align: center;
+	padding: 10px;
+	font-family: 'Arvo', serif;
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+}
+
+.grid-item h4 {
+	color: #116694;
+	font-family: 'Arvo', serif;
+	padding: 2px 16px;
+	font-size: 2rem;
+	font-weight: bold;
+}
+
+.grid-item p {
+	color: black;
+	font-family: 'Arvo', serif;
+	font-size: 1rem;
+	padding: 2px 16px;
+}
+
+.grid-item img {
+	
+	height: auto;
+	width: 100%;
+}
+
+.grid-item img:hover {
+	cursor: pointer;
+	transform: scale(1.2);
+	opacity: 0.5;
+}
+
+.grid-item button {
+	border: none;
+	outline: 0;
+	display: inline-block;
+	padding: 8px;
+	color: white;
+	background-color: #4CAF50;
+	text-align: center;
+	cursor: pointer;
+	width: 100%;
+	height: auto;
+	font-size: 1.2rem;
+}
+
+.grid-item button:hover,
+a:hover {
+	opacity: 0.7;
+}
+
+.alldogs-title {
+	color: #116694;
+	font-family: 'Arvo', serif;
+	font-size: 2rem;
+	font-weight: bold;
+	position: relative;
+	padding-left: 3rem;
+	margin-left: 10%;
+}
+
+.blog-title {
+	color: #116694;
+	font-family: 'Arvo', serif;
+	font-size: 2rem;
+	font-weight: bold;
+	position: relative;
+	margin-left: 10%;
+}
+
+.see-more-button {
+	margin: 0 auto;
+	display: block;
+	border: none;
+	background-color: lightgrey;
+	color: #116694;
+	padding: 15px 50px;
+	font-size: 1.8rem;
+	font-weight: bold;
+	cursor: pointer;
+}
+
+.footer {
+	text-align: center;
+	background: #116694;
+	padding: 2rem;
+	margin-top: 2rem;
+	color: white;
+	font-size: 2rem;
+	font-family: Arial, Helvetica, sans-serif;
+}
+
+.footer-links {
+	text-decoration: none;
+	line-height: 1.6;
+	font-size: 1.2rem;
+	font-family: Arial, Helvetica, sans-serif;
+	color: white;
+	margin: 0;
+	padding: 1rem;
+}
+
+.footer-links:hover,
+.footer-links:active {
+	color: yellow;
+}
+
+.flex-container {
+	height: 100%;
+	display: flex;
+}
+
+.box {
+	border-radius: 25px;
+}
+
+
+
+@media  (max-width: 568px) {
+    .blog-home-block  {
+		display: none;
+	
+	}
+	.header {
+		background: linear-gradient(to right, black, #00BFFF ); 
+
```

Comment:
You can probably remove these header background changes in the media queries. Since these we will want the gradient behind the image on all screen sizes.